src/Authenticator/LoginHandler.php

```diff
@@ -457,6 +457,10 @@ public function redirectAfterSuccessfulLogin(): HTTPResponse
             return $this->redirect($this->getBackURL() ?: $loginUrl);
         }
 
+        // Redirecting after successful login expects a getVar to be set, store it before clearing the session data
+        /** @see HTTPRequest::offsetSet */
+        $request['BackURL'] = $this->getBackURL();
```

Comment:
lol at this not being defined in SS3

Reply:
I'm confused; this PR is for SS4.
Is there something we need to be aware of for the SS3 backport?

Reply:
No, just that offsetSet has no logic in it in SS3. As you were